core/templates/dev/head/css/oppia.css

```diff
@@ -2788,6 +2788,59 @@ div#ng-curtain {
   margin-left: -3px;
 }
 
+/* Oppia Exploration Switch Button */
```

Comment:
Oppia exploration editor switch button.

That said, we really should stop polluting oppia.css. Can you make a standalone directive for that button and put the CSS in that file? It seems, conceptually, like it's a standalone component.